src/guide/built-ins/transition-demos/ListStagger.vue

```diff
@@ -50,11 +50,7 @@ function onLeave(el, done) {
       @enter="onEnter"
       @leave="onLeave"
     >
-      <li
-        v-for="(item, index) in computedList"
-        :key="item.msg"
-        :data-index="index"
-      >
+      <li v-for="(item, index) in computedList" :key="item.msg" :data-index="index">
```

Comment:
Separate the attributes by line so that it aligns better with the official docs 🚀 https://github.com/vuejs/docs/blob/d209fc7f1e2c3e1ffd3af9dd0da31c5f883165e0/src/guide/built-ins/transition-demos/ListStagger.vue#L53C1-L53C1